confidant/routes/identity.py

```diff
@@ -32,6 +35,26 @@ def get_client_config():
     '''
     Get configuration to help clients bootstrap themselves.
     '''
+    permissions = {
```

Comment:
ah nice, if I understand correctly, this should save us some RBAC API calls by storing these default permissions once the user logs in.

Reply:
@skiptomyliu yeah, these are the "global" permissions that apply to all views.